<?xml version="1.0"?>

<launch>
  <node name="sensor_sim_node" pkg="lx_camera_ros" type="sensor_sim_node" respawn="false" output="screen">
  <!-- -135° -->
  <param name="min_ang"          type="double"  value="-2.35619449" />
  <!-- 135° -->
  <param name="max_ang"          type="double"  value="2.35619449" />
  <param name="angle_increment"  type="double"  value="0.00582" />
  <param name="time_increment"   type="double"  value="0.00006167129" />
  <param name="range_min"        type="double"  value="0.05" />
  <param name="range_max"        type="double"  value="101.0" />
  <param name="init_range"       type="double"  value="100" />
  <!-- <remap from="scan" to="scan_back"/> -->

  <!-- 虚拟激光数据topic -->
  <param name="laser_frame_id"         type="string"  value="laser_link" />
  <param name="laser_topic_name"       type="string"  value="/sim/scan" />
  <!-- 虚拟里程计数据topic -->
  <param name="odom_frame_id"         type="string"  value="base" />
  <param name="odom_topic_name"       type="string"  value="/sim/odom" />
  <!-- 虚拟激光位姿topic -->
  <param name="laserpose_frame_id"         type="string"  value="base" />
  <param name="laserpose_topic_name"       type="string"  value="/sim/scan_pose" />

  </node>
</launch>
